### ISSUE TITLE
[A11y]: Accessibility Documentation - Update documentation for 4 teams

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#122921]
# 🔧 Accessibility Support Request

## I am requesting...

This is a ticket to track updating the accessibility documentation for the Decision Reviews / BMT 1 / BMT 2 / BMT 3 Teams 

1. The BMT & Decision Reviews GitHub Repos need 👀  to make sure they're up to date, and include links to the latest annotations and A11y 101 talks
2. The UX A11y Checklist needs updating with additional examples and links to new documentation.
3. The UX A11y Checklist docs themselves need alt text and other accessibility improvements to make them accessible to AT users
4. Check that the contents of SharePoint are up to date as well

## Links
- [DVSA Slack Canvas UX A11y Checklist](https://dsva.slack.com/docs/T03FECE8V/F09B1SN8HT6) <--- Perhaps the most used??
- [BMT Github Repo](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/claim-appeal-status/accessibility)
- [BMT UX A11y Checklist ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/claim-appeal-status/accessibility/ux-accessibility-checklist.md)
- [Decision Reviews UX A11y Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/accessibility/ux-accessibility-checklist.md)
